bugwarrior/services/linear.py

```diff
@@ -19,6 +19,10 @@ class LinearConfig(config.ServiceConfig):
     host: config.StrippedTrailingSlashUrl = config.StrippedTrailingSlashUrl(
         "https://api.linear.app/graphql", scheme="https", host="api.linear.app"
     )
+    statuses: config.ConfigList = config.ConfigList([])
+    status_types: config.ConfigList = config.ConfigList(
+        ["backlog", "unstarted", "started"]
+    )
     import_labels_as_tags: bool = False
     label_template: str = "{{label|replace(' ', '_')}}"
     also_unassigned: config.UnsupportedOption[bool] = False
```

Comment:
It would be better to enforce option compatibility constraints rather than rely on documentation. I think this would work:

```suggestion
    statuses: config.ConfigList = config.ConfigList([])
    status_types: typing.Optional[config.ConfigList] = None
    import_labels_as_tags: bool = False
    label_template: str = "{{label|replace(' ', '_')}}"
    also_unassigned: config.UnsupportedOption[bool] = False
    
    @pydantic.v1.root_validator
    def statuses_or_status_types(cls, values):
        if values["statuses"] and values["status_types"]:
            raise ValueError("statuses and status_types are incompatible")
        if not values["statuses"] and not values["status_types"]: # set default value
            values["status_types"] = ["backlog", "unstarted", "started"]
        return values
```